src/main/java/com/cs56fitnessapp/models/workout/StrengthTraining.java

```diff
@@ -0,0 +1,45 @@
+package com.cs56fitnessapp.models.workout;
+
+import com.cs56fitnessapp.models.User;
+import java.time.LocalDate;
+
+/**
+ *  @author Daniel Cervantes
+ *  @author Ksenia Koldaeva
+ *
+ * Date created: 10/18/17
+ * Last updated: 10/31/17
+ */
+
+public abstract class StrengthTraining extends Workout {
+
```

Comment:
This class should not be abstract, since we'll create instances of it and there is nothing down the inheritance chain.